app/src/main/java/com/kickstarter/libs/utils/ApplicationLifecycleUtil.kt

```diff
@@ -123,9 +138,16 @@ class ApplicationLifecycleUtil(private val application: KSApplication) :
         } else {
             if (accessToken.isNotNull() && accessToken.isNotEmpty()) {
                 client.fetchCurrentUser()
+                    .withLatestFrom(apolloClient.userPrivacy()) { user, userPrivacy ->
```

Comment:
Every time the apps comes to Foreground the current user gets updated, it was updated only with V1 information, not with anything available on GraphQL, so updating here just the bare minimum